AppController/djinn.rb

```diff
@@ -2966,7 +3105,10 @@ def restore_appcontroller_state()
 
     # Now that we've restored our state, update the pointer that indicates
     # which node in @nodes is ours
-    find_me_in_locations()
+    find_me_in_locations
```

Comment:
oh ... sorry I must have missed the memo. I was following the ruby guidelines. I can re-add them though.